Whisper.net/Native/LibraryLoader/PosixLibraryLoader.cs

```diff
@@ -0,0 +1,26 @@
+﻿// © Microsoft Corporation. All rights reserved.
+
+using System.Runtime.InteropServices;
+
+namespace Whisper.net.Native.LibraryLoader;
+
+internal class PosixLibraryLoader : ILibraryLoader
+{
+	[DllImport("libdl.so", ExactSpelling = true, CharSet = CharSet.Auto, EntryPoint = "dlopen")]
+	public static extern IntPtr NativeOpenLibraryLibdl(string filename, int flags);
+
+	[DllImport("libdl.so.2", ExactSpelling = true, CharSet = CharSet.Auto, EntryPoint = "dlopen")]
+	public static extern IntPtr NativeOpenLibraryLibdl2 (string filename, int flags);
+
+	public IntPtr OpenLibrary(string filename, int flags)
+	{
+		try
+		{
+			return NativeOpenLibraryLibdl2(filename, flags);
+		}
+		catch
```

Comment:
```suggestion
		catch (DllNotFoundException)
```
Can we add the type of the exception (in case it is crashing from other reasons like no more memory, etc, to let it throw the real exception and to not get DllNotFoundException from the second call to NativeOpenLibraryLibdl) ?

Reply:
ok thats probebly a good idea, my original reasoning is that if it fails for any reason we try the other one, but its true that for whatever reason we might not access the first one and the second one will not exists. I will update it.